backend/src/graphql/resolver/ContributionMessageResolver.ts

```diff
@@ -0,0 +1,84 @@
+import { backendLogger as logger } from '@/server/logger'
+import { RIGHTS } from '@/auth/RIGHTS'
+import { Context, getUser } from '@/server/context'
+import { ContributionMessage as DbContributionMessage } from '@entity/ContributionMessage'
+import { Arg, Args, Authorized, Ctx, Mutation, Query, Resolver } from 'type-graphql'
+import ContributionMessageArgs from '@arg/ContributionMessageArgs'
+import { Contribution } from '@entity/Contribution'
+import { ContributionMessageType } from '@enum/MessageType'
+import { ContributionStatus } from '@enum/ContributionStatus'
+import { getConnection } from '@dbTools/typeorm'
+import { ContributionMessage, ContributionMessageListResult } from '@model/ContributionMessage'
+import Paginated from '@arg/Paginated'
+import { Order } from '@enum/Order'
+
+@Resolver()
+export class ContributionMessageResolver {
+  @Authorized([RIGHTS.CREATE_CONTRIBUTION_MESSAGE])
+  @Mutation(() => ContributionMessage)
+  async createContributionMessage(
+    @Args() { contributionId, message }: ContributionMessageArgs,
+    @Ctx() context: Context,
+  ): Promise<ContributionMessage> {
+    const user = getUser(context)
+    const queryRunner = getConnection().createQueryRunner()
+    await queryRunner.connect()
+    await queryRunner.startTransaction('READ UNCOMMITTED')
+    const contributionMessage = DbContributionMessage.create()
+    try {
+      const contribution = await Contribution.findOne({ id: contributionId })
+      if (!contribution) {
+        throw new Error('Contribution not found')
+      }
```

Comment:
```suggestion
      if (!contribution) {
        logger.error(`Contribution not found!`)
        throw new Error('Contribution not found!')
      }
```

Same here as above here …
We would have more details lgged on errors.

Reply:
But this error message seems to be logged in the following catch as well … 🤔